web/src/components/Footer.tsx

```diff
@@ -25,14 +26,14 @@ export function Footer() {
           <div
             css={{
               display: 'flex',
-              width: '100%',
             }}
           >
             <Link href="/">
               <img
                 src={require('../assets/ICON.png')}
                 css={{
-                  width: '64px',
+                  width: '66px',
+                  height: '62.4px',
```

Comment:
Does a square not work (e.g. width = height)? I think the icon by itself should be a square

Reply:
No it shouldn't. The logo is 330 by 312, see https://github.com/Left-on-Read/leftonread/blob/main/web/src/assets/ICON.png

66 by 62.4 is 20% of that.